README.md

```diff
@@ -44,6 +44,11 @@ The directories (usually one, but can be multiple) where PostgreSQL's socket wil
 
 Control the state of the postgresql service and whether it should start at boot time.
 
+    postgresql_log_dir_group: "{{ postgresql_group }}"
+    postgresql_log_dir_mode: 0700
+
+Group and permissions of the `log_directory` defined in `postgresql_global_config_options`. Regarding the group, note that it's already possible to define a different group, but we don't want to use the postgresql_group already existing fact because we only want to change the log directory group, not the group for all files used by postgres (data directory, socket, config files, ...).
```

Comment:
du coup, je n'avais pas bien lu : à mon avis, tu rentres trop dans le détail de ce que tu fais dans la PR. Qqun qui lirait le readme, ne comprendrait pas toute la partie "Regarding the group, note that...". Ca c'est ok pour le body du commit, mais pas dans la doc. Perso, je retirerais toute cette phrase.

Reply:
Tu as raison ! C'est retiré et tassé.